canister/src/unstable_blocks/next_block_headers.rs

```diff
@@ -0,0 +1,291 @@
+use crate::types::BlockHash;
+use bitcoin::BlockHeader;
+use ic_btc_interface::Height;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+
```

Comment:
This file is renamed, with a small refactoring (now instead of `hash_to_height: BTreeMap<BlockHash, Height>` we have     `hash_to_height_and_header: BTreeMap<BlockHash, (Height, BlockHeader)>` in `NextBlockHeaders and some fields and comments are renamed).